templates/ehub.json

```diff
@@ -40,15 +40,24 @@
         },
         "Event Hub Resource Group": {
             "type": "String",
-            "defaultValue": ""
+            "defaultValue": "",
+            "metadata": {
+                "description": "Optional. Specifies the resource group for existing event hub. Leave empty to install new Alert Logic Event Hub infrastructure."
```

Comment:
Perhaps use Event Hub spelling throughout (instead of lower case "event hub") - all 3 descriptions.
And unify this description with the others by adding "managed" in front of "Alert Logic".